### PR TITLE
proposal of a new image docker yunohost

### PR DESCRIPTION
**Pull request by aymhce**
require a host machine with systemd and the docker image must be launched with the privileged mode

### CODE REVIEW

**Comment by Psycojoker** [comment]
This PR is a bit dying alone, it's sad :(

File: docker.md [hunk 1]
```diff
@@ -26,32 +29,42 @@ sudo pacman -Sy docker
...
+
+Please follow *Building* section [here](https://github.com/aymhce/YunohostDockerImage)
 
 ---
 
 ## Run the container
+
+> **The linux docker host must run systemd**
+
+> **WARN : --privileged mod is required for systemd**
```
Comment (zamentur) [inline docker.md]: YunoHost should detect it is install in docker to disable firewall operation. Without that if you run --priviledge, each app or domain install will redefine firewall rules of the host.

For me it is a very important point  to fix. Without I am against to speak about an official way to run yunohost with docker.

**Comment by alexAubin** [comment]
Unfortunately I kind of agree with @zamentur : if Docker's --privileged is the same as LXC's, then they are [considered unsafe](https://linuxcontainers.org/lxc/security/). While this may be suitable for tests (assuming you trust the software you're testing), it's not for production : you could imagine an app shipping some nasty code to take over the *host* of the container.

So, not sure what to do about this... While it's cool to have some documentation and an image, we can't really advertise about it. Or if we do publish this, we should clearly state it's considered unsafe for prod.

**Comment by scith** [comment]
Couldn't we create 2 docker containers instead: one volume only and one for YNH.
This way, we won't have to run the YNH container in privileged mode (I think). We can just destroy the YNH container and recreate it with extra ports opened, without losing any data.
The volume container could eventually link the guest home folder (or yunohost.backup) to the host.

?

**Comment by alexAubin** [comment]
Personally I don't understand the use case and don't realize wether there's huge demand for a docker image... It looks like this require some work to have something safe that can officially support / document ...

In the meantime, since there's not much activity on this PR, I'm flagging this as inactive. Feel free to speak up if you care / know about / want to maintain this.

**Pull request closed without merging**